src/daemons/StorageDaemon.cpp

```diff
@@ -31,6 +31,7 @@ DEFINE_string(local_ip, "", "IP address which is used to identify this server, "
 DEFINE_bool(mock_server, true, "start mock server");
 DEFINE_bool(daemonize, true, "Whether to run the process as a daemon");
 DEFINE_string(pid_file, "pids/nebula-storaged.pid", "File to hold the process id");
+DEFINE_string(clusterid_file, "data/storage.cluster.id", "File to hold the cluster id");
```

Comment:
hidden better
 but I think why to save it in rocksdb, It's safer

Reply:
Like nebula-storaged.pid, I want to let clusterid readable as a string!

Reply:
We can read it from the log or http, can it save in kvstore?  metad also do it

Reply:
Yes! Now the code has changed!